src/frontend/src/components/DownloadMetadataTemplate/index.tsx

```diff
@@ -45,6 +45,7 @@ const SampleEditTsvTemplateDownload = ({
     <CSVLink
       className={className}
       filename="metadata_template.tsv"
+      enclosingCharacter={""}
```

Comment:
So if I'm reading this correctly, we're saying that the enclosingCharacter -- for when we need to wrap something that would otherwise have a control character, like `,` in CSV -- is the empty string? So we don't ever wrap anything? I think this works in our current case because we're downloading TSVs and we're not lunatics with tabs in our data, but I am a little worried about the possibility of copy-pasting this code somewhere else and things breaking: say if someone needs to support downloading a CSV and some of the fields have commas. I think this would be good to have a comment nearby with a warning saying that enclosing characters have been completely disabled because we don't need them in this case and we need no interference with comments.